packages/icons/src/svg/star.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="12" height="12" viewBox="0 0 12 12" fill="none" stroke="#000" xmlns="http://www.w3.org/2000/svg">
-  <path d="M5.55573 2.95015C5.74203 2.58936 6.25797 2.58936 6.44427 2.95015L6.88854 2.72075L6.44427 2.95015L7.01774 4.06074C7.24115 4.49341 7.6607 4.79071 8.14297 4.85809L9.35023 5.02678C9.76718 5.08503 9.92988 5.60102 9.62176 5.88791L8.80845 6.64516C8.43523 6.99265 8.26412 7.50595 8.35418 8.00788L8.55373 9.12011C8.62649 9.52562 8.2051 9.84018 7.83701 9.65514L6.67373 9.07034C6.24985 8.85725 5.75015 8.85725 5.32627 9.07034L4.16299 9.65514C3.7949 9.84019 3.37351 9.52563 3.44627 9.12011L3.64583 8.00788C3.73588 7.50595 3.56477 6.99265 3.19155 6.64516L2.37824 5.88791C2.07012 5.60102 2.23282 5.08503 2.64977 5.02678L3.85703 4.85809C4.3393 4.79071 4.75885 4.49341 4.98226 4.06074L5.55573 2.95015Z" />
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This svg can be optimized using
https://jakearchibald.github.io/svgomg/

Example

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path fill-rule="evenodd" d="M12 1a1 1 0 0 1 .897.557l2.857 5.79 6.39.934a1 1 0 0 1 .554 1.705l-4.623 4.503 1.09 6.362a1 1 0 0 1-1.45 1.054L12 18.9l-5.715 3.005a1 1 0 0 1-1.45-1.054l1.09-6.362-4.623-4.503a1 1 0 0 1 .553-1.705l6.39-.935 2.858-5.789A1 1 0 0 1 12 1Zm0 3.26L9.807 8.702a1 1 0 0 1-.752.546l-4.907.718 3.55 3.457a1 1 0 0 1 .288.885l-.838 4.883 4.387-2.307a1 1 0 0 1 .93 0l4.387 2.307-.838-4.883a1 1 0 0 1 .288-.885l3.55-3.457-4.907-.718a1 1 0 0 1-.752-.546L12 4.259Z" clip-rule="evenodd"/>
</svg>
```